components/brave_wallet/browser/brave_wallet_prefs.cc

```diff
@@ -248,6 +248,13 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterDictionaryPref(kBraveWalletMnemonic);
   registry->RegisterBooleanPref(kBraveWalletLegacyEthSeedFormat, false);
   registry->RegisterBooleanPref(kBraveWalletMnemonicBackedUp, false);
+
+  // Register Deprecated CryptoWallet prefs
+  // We can eventually remove these. Code removed 05/2025
+  registry->RegisterIntegerPref(kERCPrefVersionDeprecated, 0);
+  registry->RegisterStringPref(kERCAES256GCMSivNonceDeprecated, "");
+  registry->RegisterStringPref(kERCEncryptedSeedDeprecated, "");
+  registry->RegisterBooleanPref(kERCOptedIntoCryptoWalletsDeprecated, false);
 }
```

Comment:
nit:
It was wrapped with `#if BUILDFLAG(ETHEREUM_REMOTE_CLIENT_ENABLED)` ( which now should be `#if BUILDFLAG(BUILDFLAG(ENABLE_EXTENSIONS)`)

Reply:
Actually it should not break anything on mobile platforms, so I'm ok to leave it as is

Reply:
OK, I'll leave it as is